### PR TITLE
Document system requirements

### LINKED ISSUE #399: add: Minimum Hardware Requirements
I would add `Minimum Hardware Requirements` section or page.

This article mentions it, but it could be outdated:
https://etherworld.co/2021/03/11/how-to-become-an-eth2-validator-on-teku/#minimumhardwarerequirements

Other clients with this info in their doc:
EL:
- https://docs.nethermind.io/nethermind/first-steps-with-nethermind/system-requirements
- https://besu.hyperledger.org/en/stable/public-networks/get-started/system-requirements/ 
- https://github.com/ledgerwatch/erigon#system-requirements
- https://geth.ethereum.org/docs/interface/hardware 

CL:
- https://lighthouse-book.sigmaprime.io/system-requirements.html
- https://chainsafe.github.io/lodestar/#specifications
- https://nimbus.guide/hardware.html
- https://docs.prylabs.network/docs/install/install-with-script/#step-1-review-prerequisites-and-best-practices

### PR DESCRIPTION
**Pull request by alexandratran**
Document system requirements for Teku. Fixes #399.

Preview: https://doc-teku-g6dvqfxm6-consensys-ddffed67.vercel.app/development/get-started/system-requirements

### CODE REVIEW

**Review by Magkoooh — APPROVED** [review]

**Comment by lucassaldanha** [comment]
As a starting point, I'd say the minimum requirements for running a full node (Beacon Node + Validator):

CPU: 4+ cores @ 2.8+ GHz
Memory: 16GB+ RAM
Storage: SSD with at least 2TB free space

(similar to Prysm)

File: docs/get-started/system-requirements.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# System requirements
+
+The following are the minimum system requirements for running a full node, with Teku as a beacon node and validator client:
+
+- CPU: 4 cores at 2.8 GHz
+- Memory: 16 GB RAM
+- Storage: SSD with 2 TB free space
+- Operating system: 64-bit Linux, Mac OS X, or Windows
+- Internet: Broadband internet connection (10 Mbps)
+- Power: Uninterruptible power supply (UPS)
```
Comment (lucassaldanha) [inline docs/get-started/system-requirements.md:16]: UPS is a bit extreme. The way the inactivities penalties work means that even if you go offline for a couple of hours, you can quickly recoup that being online later. The only tragedy happens if you are offline when it is time to propose a block or during a sync committee participation, but both are very rare.

So maybe we add a note that this is recommended but not a requirement?

Comment (lucassaldanha) [inline docs/get-started/system-requirements.md:16]: UPS is a bit extreme. The way the inactivities penalties work means that even if you go offline for a couple of hours, you can quickly recoup that being online later. The only tragedy happens if you are offline when it is time to propose a block or during a sync committee participation, but both are very rare.

So maybe we add a note that this is recommended but not a requirement?

**Review by lucassaldanha — APPROVED** [review]

**Review by bgravenorst — APPROVED** [review]
lgtm

**Pull request merged**